### PR TITLE
Kashish okcupid

### PR DESCRIPTION
**Pull request by kashishoberoi**

### CODE REVIEW

File: src/content-acquisition/okcupid/MatchScrape.py [hunk 1]
```diff
@@ -86,12 +88,16 @@ def retreive_data(count,driver,data_dict,soup):
         data_dict['background'] = items[1].text
     
     img = soup.find('img', class_='active')['src']
-    download_img(img,data_dict['name'].strip()+str(i)+'.jpeg',path2)
+    img_filename=data_dict['name'].strip()+str(i)+'.webp'
+    download_img(img,img_filename,path2)
+    im_loc=path2+"/"+img_filename
+    im = Image.open(im_loc).convert("RGB")
+    im.save(path2+"/"+data_dict['name'].strip()+str(i)+'.jpg',"jpeg")
+    os.remove(im_loc)
```
Comment (AmoghM) [inline src/content-acquisition/okcupid/MatchScrape.py]: what is os.remove() required?

**Comment by AmoghM** [comment]
please remove config/tinder.yaml

File: src/content-acquisition/okcupid/MatchScrape.py (hunk 1 above)
Comment (kashishoberoi) [reply src/content-acquisition/okcupid/MatchScrape.py]: Its used to remove the webp image

**Comment by AmoghM** [comment]
1. Please copy the .gitignore from the master. We don't want configs checked in.
3. chromedriver path hardcoded. Please put it in config
3. Add ReadMe for chromedriver installation and config example
4. Fix bug. Line 118: url_match is a variable with no value

**Pull request closed without merging**